Sources/Fullscreen/FrontpageSearch/FrontpageSearchView.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+import FinniversKit
+import Foundation
+
+public protocol FrontpageSearchViewDelegate: AnyObject {
+    func frontpageSearchView(didSelectFavoriteButton button: UIButton, forAdWithId: Int, cell: FrontpageSearchImageResultCollectionViewCell)
+    func frontpageSearchView(_ view: FrontpageSearchView, didSelectResultAt indexPath: IndexPath, uuid: UUID)
```

Comment:
Is there a need to pass the `uuid` to the delegate?

Reply:
Nevermind, I see it's being used from the app

Reply:
UUID ❤️  😂